modules/python/python-dest.c

```diff
@@ -119,9 +119,9 @@ python_dd_format_stats_instance(LogThreadedDestDriver *d)
   static gchar persist_name[1024];
 
   if (d->super.super.super.persist_name)
-    g_snprintf(persist_name, sizeof(persist_name), "python,%s", d->super.super.super.persist_name);
+    g_snprintf(persist_name, sizeof(persist_name), PYTHON_MODULE_PERSIST_KEY ",%s", d->super.super.super.persist_name);
```

Comment:
Do we need to change persist? Simply having `python` in any case looks neat to me.

Reply:
It's hard to tell. In my opinion, persist keys should be as unique as possible. We fortunately have key collision detection in OSE, but a Python 2 module and Python 3 module is different in my mind. Upgrading a source/destination/parser from v2 to v3 or backwards has to be a deliberate action, I'd force the user to use `persist-tool` or `persist-name()` to make the transition if they need the old persist entry.

Reply:
One last argument that might worth considering is inconsistency. If I see a persist name of `python`, I cannot tell if it comes from
- `--with-python=2 --disable-python2_and_3`
- `--with-python=3 --disable-python2_and_3`
- `--enable-python2_and_3` but python2 is used.

Currently `PYTHON_MODULE_PERSIST_KEY` persist does not provide uniqueness among python versions. To provide uniqueness, you need to eliminate `python`, and provide `python2` and `python3` consistently.

Reply:
Yes, this is for compatibility. I'd like to remove the support of Python 3 with the unversioned python keyword as soon as possible.